src/index.css

```diff
@@ -20,6 +20,14 @@
 	margin: 0 auto;
 }
 
+html {
+	overflow-y: scroll;
+}
+
+body {
+	margin: 0;
```

Comment:
I dont know if it is better to have a TW class on body or this. But I would expect tailwind to reset it by default 🤷